includes/admin/admin-actions.php

```diff
@@ -391,9 +391,143 @@ function _give_show_test_mode_notice_in_admin_bar( $wp_admin_bar ) {
 
 	return true;
 }
+
 add_action( 'admin_bar_menu', '_give_show_test_mode_notice_in_admin_bar', 1000, 1 );
 
 /**
+ * Add Link to Import page in from donation archive and donation single page
+ *
+ * @since 1.8.13
+ */
+function give_import_page_link_callback() {
+	?>
+    <a href="<?php echo esc_url( give_import_page_url() ); ?>"
+       class="page-import-action page-title-action"><?php esc_html_e( 'Import Donations', 'give' ); ?></a>
```

Comment:
@raftaar1191 unnecessary `esc_html`. Remove it.

Reply:
@ravinderk 
I was reading this article that why I used it. Please have a look into it
https://vip.wordpress.com/documentation/validating-sanitizing-escaping/#always-escape-late

Reply:
This is OK @raftaar1191

Reply:
@raftaar1191 In case string is hard coded, not user defined

Reply:
Done